plugins/3scale-backend/src/module.ts

```diff
@@ -36,7 +35,7 @@ export const catalogModule3ScaleEntityProvider = createBackendModule({
       async init({ catalog, config, logger, scheduler }) {
         catalog.addEntityProvider(
           ThreeScaleApiEntityProvider.fromConfig(config, {
-            logger: loggerToWinstonLogger(logger),
+            logger: logger,
```

Comment:
```suggestion
            logger,
```
Please change every instance of longhand to shorthand.